lib/AST/DeclContext.cpp

```diff
@@ -518,7 +518,10 @@ ResilienceExpansion DeclContext::getResilienceExpansion() const {
       // we serialize the parent's body.
       if (AFD->getDeclContext()->isLocalContext())
         continue;
-      
+
+      if (AFD->isInvalid())
```

Comment:
Heh. I have a lot of Opinions here, the foremost being "picking a resilience abstraction at the AST level is a bad idea" and the next being "calling getEffectiveContext at the AST level is also a bad idea".

I don't actually care what predicate we're checking, but there really are always going to be cases where some AST queries are going to be inaccurate. getEffectiveContext requires having a valid DeclContext chain all the way up to the module, because of the rule that a public entity within an internal context is effectively internal.

We might be able to set up "valid" accessibility for an initializer outside of a nominal type, but it certainly won't have a valid type.